File: new/atoms/tz-stale-days.test.mjs
import * as m from './tz-stale-days.mjs';
const SNAP = {"TZ_STALE_DAYS":"90"};
let f = 0;
for (const [k, s] of Object.entries(SNAP)) if (JSON.stringify(m[k]) !== s) { console.error('✗ ' + k + ' סטה מהצילום'); f = 1; }
if (f) process.exit(1); console.log('✓ tz-stale-days: צילום-ערך תואם — ירוק');

File: new/dart-maor/tz-stale-days_test.dart
// בדיקת-חוזה (רתמת-זהב) · tz-stale-days — צילום-ערך jsonEncode≡JSON.stringify.
// חוללה מכנית (gen-data-dart). הרצה: dart run --enable-asserts ⇒ OK
import 'dart:convert';
import 'tz-stale-days.dart';

void main() {
  // ‏1) צילום-ערך: jsonEncode ≡ JSON.stringify של המקור (זהה-ביט).
  if (jsonEncode(tzStaleDays) != '90') {
    throw StateError('FAIL tzStaleDays: צילום-הערך סטה');
  }
  print('OK tz-stale-days: 1 data-snapshot(s) passed');
}
